src/lib/random/random.c

```diff
@@ -58,7 +52,7 @@ uint64_t RandomU64(void)
 }
 
 /**
- * This function returns a random number according to an Exponential distribution.
+ * Returns a random number according to an Exponential distribution.
```

Comment:
I think comments should be "Return", rather than "Returns", in general.

Reply:
Ok, it will be changed!